test/versioned/mongodb/cursor-duration.tap.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
```

Comment:
Given a new file, prob 2021?

Reply:
I'm unclear which way we should go here. I've been told both current year and year we started copyrighting files(2020)